frontends/web/src/routes/settings/components/advanced-settings/enable-auth-setting.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ChangeEvent, Dispatch } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Toggle } from '../../../../components/toggle/toggle';
+import { SettingsItem } from '../settingsItem/settingsItem';
+import { TBackendConfig, TConfig } from '../../advanced-settings';
+import { setConfig } from '../../../../utils/config';
+import { subscribeAuthOk, triggerAuth } from '../../../../api/backend';
+
+type TProps = {
+  backendConfig?: TBackendConfig;
+  onChangeConfig: Dispatch<TConfig>;
+}
+
+export const EnableAuthSetting = ({ backendConfig, onChangeConfig }: TProps) => {
+  const { t } = useTranslation();
+
+  const handleToggleFee = async (e: ChangeEvent<HTMLInputElement>) => {
+    // force a last auth before disabling it
```

Comment:
shouldn't we also do an auth for enabling it, to test that it works? say the auth thing does not work for some reason on some systems or Android versions - if we enabled this the user might not be able to unlock it ever again.

Reply:
Good point. I did it, but since the backend automatically skipped the auth if the config was set to false, I had to introduce a new concept of forced auth, which is executed without considering the authentication config flag.